lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -183,34 +183,53 @@ const Reportback = {
    * @param  {array} data  Array of Kudos objects from API.
    * @return {array}
    */
-  getReactions: function(data) {
-    let reactions = [];
-
-    forEach(this.reactions.approved, (term) => {
-      const termId = this.reactions.terms[term];
+  getReactions: function(data, reactionFromRogue) {
+    if (reactionFromRogue) {
```

Comment:
shoot now this is broken if rogue is turned off with new code :/ Here's the error: 
```
Uncaught TypeError: Cannot read property 'length' of undefined
    at eval (eval at <anonymous> (app.js:14415), <anonymous>:23:36)
    at Object.templatize (app.js:17455)
    at Object.<anonymous> (app.js:17415)
    at fire (lib.js:3282)
    at Object.fireWith [as resolveWith] (lib.js:3394)
    at done (lib.js:9448)
    at XMLHttpRequest.callback (lib.js:9852)
``` 
I was also getting this before I added the extra code to check if it's from Rogue and `likedByCurrentUser`